wcc-contentful-app/spec/wcc/contentful_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe WCC::Contentful do
+  # describe '.validate_models!' do
```

Comment:
still need this?

Reply:
I want to move it into a spec for the rake task, but that should really be part of #40